src/app/modules/report-module/index.tsx

```diff
@@ -559,28 +567,39 @@ export default function ReportModule() {
     return hasTextValue || framesArrayState;
   }, [reportName, framesArray, headerDetails]);
 
+  const canChartEditDelete = React.useMemo(() => {
+    return isAuthenticated && reportGetData?.owner === user?.sub;
+  }, [user, isAuthenticated, reportGetData]);
+
+  const inEditAndCanEdit = view === "edit" ? canChartEditDelete : true;
```

Comment:
variable name should be related to the purpose for which it will be used. In this case can be more specific to report header spacer div thingy